nephos/mail_notifier.py

```diff
@@ -69,13 +69,13 @@ def send_mail(msg, msg_type):
 
 def add_to_report(msg):
     """
-    Append string to the EOF of report file.
+    Append string to the EOF of report files.
```

Comment:
Well there are many instances where the changes made do not improve the docs; rather they create a false meaning. One such instance is this only. We only have a single report file and hence the text.

You've done a great job but I would like you to again go through things; inquire with me and clarify doubts and then make the amendments.